Model/ModelManager.php

```diff
@@ -376,7 +386,7 @@ public function getNormalizedIdentifier($entity)
         }
 
         // the entities is not managed
-        if (!$entity /*|| !$this->getEntityManager($entity)->getUnitOfWork()->isInIdentityMap($entity) // commented for perfomance concern */) {
+        if (!$entity || !$this->getEntityManager($entity)->getUnitOfWork()->isInIdentityMap($entity)) {
```

Comment:
// persist if the form was valid and if in preview mode the preview was approved
    if ($isFormValid && (!$this->isInPreviewMode() || $this->isPreviewApproved())) {
        try {
            // still in UOW here
            $object = $this->admin->update($object);
            // not in UOW here